net/speedtest-go/Makefile

```
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Copyright (C) 2021 ImmortalWrt.org
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=speedtest-go
+PKG_VERSION:=1.6.10
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/showwin/speedtest-go/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=3f2d6ae3b60d8b6e972038fd2e13f3fdeca75647d88785d65bfa0d99a2f4fd79
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
```

```suggestion
PKG_LICENSE_FILES:=LICENSE
PKG_MAINTAINER:=TeleostNaCl Dai <teleostnacl@gmail.com>
```

as the PR description please add yourself as the maintainer
or you can assign me